File: src/main/java/in/dragonbra/javasteam/util/compat/ObjectsCompat.java
package in.dragonbra.javasteam.util.compat;

/**
 * @author steev
 * @since 2018-03-21
 */
public class ObjectsCompat {
    public static boolean equals(Object a, Object b) {
        return (a == b) || (a != null && a.equals(b));
    }
}

File: src/main/java/in/dragonbra/javasteam/util/compat/ObjectsCompat.kt
package `in`.dragonbra.javasteam.util.compat

/**
 * Compatibility for [java.util.Objects] for Android, which requires API 19+.
 * @author steev
 * @since 2018-03-21
 */
object ObjectsCompat {
    @JvmStatic
    fun equals(a: Any?, b: Any?): Boolean = a == b
}
